website/source/docs/builders/vmware-iso.html.md.erb

```diff
@@ -82,6 +82,8 @@ builder.
 -   `iso_url` (string) - A URL to the ISO containing the installation image.
     This URL can be either an HTTP URL or a file URL (or path to a file). If
     this is an HTTP URL, Packer will download it and cache it between runs.
+    Packer uses hashicorp/go-getter in file mode in order to perform a download.
+    See usage of [hashicorp/go-getter](https://github.com/hashicorp/go-getter).
```

Comment:
Is it okay to link go-getter docs here, or should I instead describe what can packer do ?